specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json

```diff
@@ -10875,7 +10876,8 @@
           "type": "string",
           "description": "The name of the service.",
           "enum": [
-            "AutomaticRepairs"
+            "AutomaticRepairs",
+            "DELETEME"
```

Comment:
>DELETEME [](start = 13, length = 8)

Why do we need this here and then why do we remove it in the .md file?

Reply:
Why is that a problem? If it's always the same value, why are we forcing users to specify it?

Reply:
We expect more enum values to get added in future, and that will be a breaking change in SDK. So asking users to specify "serviceName" always is the right thing to do to avoid such breaking change.

Reply:
If you make the current value the default in a future API version when the property is not specified, it won't be a breaking change. (Code written against an old SDK will continue working after upgrade.)

The shape of the API isn't really affected by this, so I'll sign off, but this is a really unusual pattern. I will defer to @fengzhou-msft 's judgment here as I'm not an autorest expert and can't comment if this will work correctly in all languages.

Also keep in mind that some users generate their own clients from our swaggers with tooling that may not understand the readme.md directive. Those users will see the dummy DELETEME value :(

Reply:
I've dropped it from the swagger, and just used the configuration to dynamically insert it and then remove it.